app/editor/src/features/navbar/NavBar.tsx

```diff
@@ -7,8 +7,40 @@ import { Row } from 'tno-core/dist/components/flex/row';
  */
 export const NavBar: React.FC = () => {
   const [activeHover, setActiveHover] = useState<string>('');
+
+  const [hide, setHide] = useState<boolean>(false);
+  const hideRef = useRef(hide);
+  hideRef.current = hide;
```

Comment:
If you're using a ref is there any reason to use state?  Why not just `hideRef.current = false` or `hideRef.current = true`?

Reply:
Good question. Yeah it seems no need to use state in this case, and the code is updated again. Thanks